PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/InterstitialAdUnit.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  InterstitialAdUnit.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import UIKit
+
+public class InterstitialAdUnit: PBMBaseInterstitialAdUnit {
+
+    @objc public override init(configId: String) {
+        super.init(configId: configId, eventHandler: PBMInterstitialEventHandlerStandalone())
+    }
+
+    @objc public override init(configId: String, minSizePercentage: CGSize) {
+        super.init(
+            configId: configId,
+            minSizePercentage: minSizePercentage,
+            eventHandler: PBMInterstitialEventHandlerStandalone())
+    }
+
+    @objc public override init(configId: String, minSizePercentage:CGSize, eventHandler: Any) {
+        super.init(configId: configId, minSizePercentage: minSizePercentage, eventHandler: eventHandler)
+    }
+    
+    override init(configId:String, minSizePerc: NSValue?, eventHandler:Any?) {
+        super.init(configId: configId, minSizePerc:minSizePerc, eventHandler:eventHandler)
+    }
+    
+    
+    // MARK: - Protected overrides
+
+    @objc public override func callDelegate_didReceiveAd() {
+        let delegate = self.delegate as? PBMInterstitialAdUnitDelegate
```

Comment:
this code construction requires let or guard. The current one looks strange though it makes sense.